### PR TITLE
 Feat: swap Exact Amount Out: Load testing

### PR DESCRIPTION
**Pull request by deividaspetraitis**
This pull request adds load testing tasks for exact amount out token swap quote. Tasks mimics existing ones for exact amount in.

## Summary by CodeRabbit

- **New Features**
	- Introduced flexible token handling in the API, allowing optional parameters for token denominations.
	- Added support for exact output amount quotes in the router functionality.
	- Enhanced testing infrastructure with new utility functions for managing routable pools and quotes.

- **Bug Fixes**
	- Improved error handling across various components to provide clearer feedback.

- **Documentation**
	- Updated API documentation to reflect changes in parameter requirements and new functionalities.

- **Chores**
	- Enhanced code readability and organization across multiple files without affecting functionality.

### CODE REVIEW

**Comment by p0mvn** [comment]
Do you mind please posting a screenshot of the locust UI when only the "in given out" tests are enabled (the rest can be commented out for the screenshot scope)?

I would like to see the RPS and p95

**Comment by deividaspetraitis** [comment]
> Do you mind please posting a screenshot of the locust UI when only the "in given out" tests are enabled (the rest can be commented out for the screenshot scope)?
> 
> I would like to see the RPS and p95

Please see [a link ](https://filebin.net/ahsgql986tgwu84f) to report files, Github seems does not support uploading .html files. Both endpoints were tested under same conditions:

- Number of users: 4
- Ramp up: 100 

```
docker compose up --scale worker=4
```

----

Please note that I have made a few more commits since your last review with efforts to separate tests to make the running independently of each other a bit easier. For example:

```
    command: -f /mnt/locust/locustfile.py Pools Routes --master -H http://....
```

**Comment by p0mvn** [comment]
> a link

Nice - thank you. Results look better than I expected

File: locust/token_prices.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+    @task
+    def tokenPrices(self):
+        bases = ""
+        for i in range(len(top10ByVolumePairs)):
+            bases += top10ByVolumePairs[i]
+            if i < len(top10ByVolumePairs) - 1:
+                bases += ","
+
+        self.client.get(f"/tokens/prices?base={bases}")
```
Comment (p0mvn) [inline locust/token_prices.py:21]: Thank you for this clean up and separation into separate files!

**Review by p0mvn — APPROVED** [review]
Nice work!

**Comment by p0mvn** [comment]
Please feel free to merge once ready 🙏

**Pull request merged**